unittest/gunit/keyring_vault/uuid.h

```diff
@@ -0,0 +1,22 @@
+#include <string>
+#include <boost/uuid/uuid.hpp>            // uuid class
+#include <boost/uuid/uuid_generators.hpp> // generators
+#include <boost/uuid/uuid_io.hpp>         // streaming operators etc.
+
+static std::string generate_uuid()
+{
+  boost::uuids::uuid uuid = boost::uuids::random_generator()();
```

Comment:
creating generator itself on each call can be CPU consuming. I would initialize it once and put it in a static variable in this file.

Reply:
Done.